doc/lsst.afw.multiband/multiband.rst

```diff
@@ -0,0 +1,766 @@
+    #  [ 1.  1.  1.  1.  1.]]
+
+    # [[ 2.  2.  2.  2.  2.]
+    #  [ 2.  2.  2.  2.  2.]
+    #  [ 2.  2.  2.  2.  2.]]]
+
+Notice that the `XY0` (or minimal bounding box coordinate) has been subtracted
+from the spatial indices. So the same subset can also be extracted with
+all of the following methods:
+
+.. code-block:: python
+
+    subset = mImage[1:3, :1005, :2003]
+    subset = mImage[1:3, :-194, :-96]
+    subset = mImage[1:3, Box2I(Point2I(1000, 2000), Extent2I(5, 3))]
+
+.. warning::
+    Negative indices can only be used in a pythonic fashion if `XY0 >= 0`,
+    otherwise negative indices are interpreted to be negative coordinates.
+
+However notice if we use indices less than `XY0` we get an error:
+
+.. code-block:: python
+
+    subset = mImage[1:3, 1:1006, 1:2004]
+
+    # Output
+    #---------------------------------------------------------------------------
+    #IndexError                                Traceback (most recent call last)
+    #<ipython-input-25-9adb2171fc95> in <module>()
+    #----> 1 subset = mImage[1:3, 1:1005, 1:2003]
+    #
+    #~/lsst/code/afw/python/lsst/afw/multiband.py in __getitem__(self, args)
+    #    203             return result
+    #    204 
+    #--> 205         return self._slice(filters=filters, filterIndex=filterIndex, indices=indices[1:])
+    #    206 
+    #    207     def filterToIndex(self, filterIndex):
+    #
+    #~/lsst/code/afw/python/lsst/afw/image/multiband.py in _slice(self, filters, filterIndex, indices)
+    #    291         if len(indices) > 0:
+    #    292             allSlices = [filterIndex, slice(None), slice(None)]
+    #--> 293             sy, sx = self.imageIndicesToNumpy(indices)
+    #    294             if sy is not None:
+    #    295                 allSlices[-2] = sy
+    #
+    #~/lsst/code/afw/python/lsst/afw/multiband.py in imageIndicesToNumpy(self, indices)
+    #    300 
+    #    301             if sx is not None:
+    #--> 302                 sx = self._removeOffset(sx, x0, bbox.getMaxX())
+    #    303             if sy is not None:
+    #    304                 sy = self._removeOffset(sy, y0, bbox.getMaxY())
+    #
+    #~/lsst/code/afw/python/lsst/afw/multiband.py in _removeOffset(self, index, x0, xf)
+    #    343                 start = None
+    #    344             else:
+    #--> 345                 start = _applyBBox(index.start, x0, xf)
+    #    346             if index.stop is None:
+    #    347                 stop = None
+    #
+    #~/lsst/code/afw/python/lsst/afw/multiband.py in _applyBBox(index, x0, xf)
+    #    329             if index > 0 and (index < x0 or index > xf):
+    #    330                 err = "Indices must be <0 or between {0} and {1}, received {2}"
+    #--> 331                 raise IndexError(err.format(x0, xf, index))
+    #    332             newIndex = index - x0
+    #    333             if index < 0:
+    #
+    #IndexError: Indices must be <0 or between 1000 and 1199, received 1
+    #
```

Comment:
Sorry, I thought I made a note when I asked you to look over this again but it looks like I didn't. I updated everything except the docs, just in case there are any more API changes. I have a jupyter notebook that contains all of the statements in the docs and will re-run them all (to make sure that the docs are up to date) before merging.